.ado/variables/vs2019.yml

```diff
@@ -1,6 +1,6 @@
 variables:
   VmImage: windows-2019
   MSBuildVersion: 16.0
-  GoogleTestAdapterPath: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\Extensions\ked32bft.vu0'
+  GoogleTestAdapterPathExpression: '(Get-ChildItem "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\Extensions\" -Recurse -Include GoogleTestAdapter.Core.dll).Directory.FullName'
```

Comment:
normally it isn't guaranteed that the VS install is on the C drive (or even in program files (x86))
ideally we'd use vswhere but otherwise can we use `${env:ProgramFiles(x86)}` ?

Reply:
Good idea. I'll add this.
Tests are passing now.